### ISSUE TITLE
DCC support?

### ISSUE THREAD

**Issue by 097115** [thimc/tinyirc#2]
Probably, no plans for it? :)

**Comment by thimc** [comment]
No, can't say that I have. To be honest I have never heard of it before nor do I know how it works, as ignorant as that may be.

**Comment by 097115** [comment]
Thanks for your comments! (And feel free to close this issue :)
